### ISSUE TITLE
FP raised by S4670 on "foreignObject" selector

### ISSUE THREAD

**Issue by nicolas-harraudeau-sonarsource** [SonarSource/sonar-css#206]
[S4670](https://jira.sonarsource.com/browse/RSPEC-4670) raises a False Positive on the [`foreignObject`](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/foreignObject) selector which is used in SVG.
Reproducer:
```css
.test foreignObject { /* Compliant  but raising an issue*/
  color: white
}
```

In the meantime the workaroud is to add `foreignObject` to the "ignoreTypes" parameter of the rule. Ex: `/(^mat-)|foreignObject/`

**Comment by vilchik-elena** [comment]
Reported to stylelint https://github.com/stylelint/stylelint/issues/4472
